dojson/contrib/to_marc21/fields/bd80x83x.py

```diff
@@ -332,7 +332,5 @@ def reverse_series_added_entry_uniform_title(self, key, value):
         '8': utils.reverse_force_list(
             value.get('field_link_and_sequence_number')),
         '$ind1': '_',
-        '$ind2': value.get(
-            'nonfiling_characters',
-            '_'),
+        '$ind2': value.get('nonfiling_characters') if value.get('nonfiling_characters') in valid_nonfiling_characters else '_',
```

Comment:
Sorry for the stupid question, but where else do we validate the data ? I know that we have JSON Schemas, but doing validation here is cheap and doJSON and JSON Schemas can desynchronize (I know they shouldn't, but better safe than sorry :wink:  ).
I though that removing data that doesn't fit the schema is a correct way to go.